marketplaceItems/avatarCustomization101/appResources/appData/resources/css/app.css

```diff
@@ -0,0 +1,210 @@
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: 'Cairo', sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #005916;
+    color: #666666;
+  }
+
+  table {
+      width: 100%;
+      font-size: 16px;
+  }
+
+  p {
+      margin-bottom: 0;
+  }
+
+  h1, h2, h3, h4, h5, h6 {
+    font-family: 'Raleway', sans-serif;
+  }
+
+  h3 {
+    margin: 10px 0;
+  }
+  
+  .flex-grow-1 {
+      flex-grow: 1;
+  }
+
+  .flex-grow-2 {
+      flex-grow: 2;
+  }
+
+  .flex-grow-3 {
+      flex-grow: 3;
+  }
+
+  .nav-tabs {
+    border: none;
+  }
+
+  .active.nav-item.nav-link {
```

Comment:
i'm looking through this file and noticing a lot of CSS classes that aren't in use in your HTML. could you please look through your CSS and remove all unused styles?

Reply:
after you do this, let me know and i will look through the rest of the file.